setup.cfg

```diff
@@ -31,12 +31,14 @@ install_requires =
     invenio-records-resources>=5.0.0,<6.0.0
     lxml>=4.5.0
     PyYAML>=5.4.1
+    oaipmh-scythe @ git+https://github.com/ulbmuenster/invenio-oaipmh-scythe.git
```

Comment:
Do we want to release `invenio-oaipmh-scythe` to PyPI before merging or are we happy to keep a Git dependency?

Reply:
I thought so, too, but it's just a temporary solution until we completely get rid of Python 3.9.